Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_015.AkamaiCredentialsValidator.cs

```diff
@@ -60,9 +60,11 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
             string id = fingerprint.Id;
             string host = fingerprint.Host;
             string secret = fingerprint.Secret;
+            string asset = secret.Truncate(); // TBD
```

Comment:
>asset [](http://example.com/codeflow?start=19&length=5)

there may be a better option for this security model but someone would need to get it into the debugger to take a look. that would be a good thing to do to learn about the subtleties in this change. any takers?